demo/x-key-aware.html

```diff
@@ -17,10 +18,30 @@
       display: block;
       position: relative;
     }
+
+    pre {
+      color: var(--google-blue-700);
+    }
+    
+    .keys {
+      line-height: 25px;
+    }
+
+    .keys span {
+      cursor: default;
+      background-color: var(--google-grey-100);
+      border: 1px solid var(--google-grey-300);
+      padding: 1px 5px;
+      border-radius: 5px;
+    }
   </style>
   <template>
-    <span>Press any of these keys: <span>[[boundKeys]]</span></span>
-    <pre id="output">[[pressed]]</pre>
+    <p class="keys">Press any of these keys:
```

Comment:
I think it would be nicer/consistent with all the other demos if this content lived in the white-background-with-shadow-card-looking thing. There's styles for this in `paper-styles/demo-pages`. So if you look, for example, at the `iron-autogrow-textarea`, this should be

```
<div class="vertical-section-container centered">
      <h4>Press any of these keys</h4>
        <div class="vertical-section">
          <template is="dom-repeat" items="..".>
           ...
          </template>
       </div>
   </div>
```